apps/ws/package.json

```diff
@@ -10,7 +10,7 @@
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nodemon",
     "start:dev": "nodemon",
-    "start:test": "cross-env NODE_ENV=test TZ=UTC nodemon",
+    "start-test": "cross-env NODE_ENV=test TZ=UTC nodemon",
```

Comment:
@djabarovgeorge bahh so strange this not working with nx and now forces us to change the naming in places and loose consistency. Maybe worth opening an issue in the nx github repo? or stacoverflow for this? I'm checking with some friends aswell.

Reply:
Yes, I could not agree more I will look into it some more.

Reply:
Managed to make it work with (maybe there is a better way for this):
```
 nx run-many --target=start:test --projects=@notifire/api
```

Reply:
Nice! Awesome work around!!  😁